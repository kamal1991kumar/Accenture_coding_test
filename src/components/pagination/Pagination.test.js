import React from "react";
import { render } from "@testing-library/react";
import Pagniation from "./index";

const props = {
  start: 0,
  total: 10,
  threshold: 10,
};

describe("Pagniation Component", () => {
  test("Should render on screen", () => {
    const { container } = render(<Pagniation {...props} />);
    expect(container).toMatchSnapshot();
  });

  test("Should match the pagination text", () => {
    const { getByTestId } = render(<Pagniation {...props} />);
    expect(getByTestId("pagination").textContent).toBe("10 out of 10");
  });
});
