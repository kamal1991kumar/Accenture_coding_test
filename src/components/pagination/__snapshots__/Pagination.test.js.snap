// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagniation Component Should render on screen 1`] = `
<div>
  <div
    class="pagination"
    data-testid="pagination"
  >
    10
     out of 
    10
  </div>
</div>
`;
